project/Build.scala

```diff
@@ -982,8 +982,6 @@ object Build {
       libraryDependencies +=
         "org.scala-lang" % "scala-reflect" % scalaVersion.value % "provided",
 
-      // js.JSApp is annotated with @JSExportDescendentObjects
-      scalacOptions += "-P:scalajs:suppressExportDeprecations",
```

Comment:
Looks like this change is not in the right commit. It should be in the same commit where we remove `@JSExportDescendentObjects` from `JSApp`.